src/main/resources/perseo-core.properties

```diff
@@ -25,4 +25,5 @@ action.url = http://127.0.0.1:9090/actions/do
 
 # Time in milliseconds (long) to "expire" a "dangling" rule
 rule.max_age= 60000
-
+properties.path = /etc/perseo-core.properties
```

Comment:
Why this is needed? I mean, if you already are parsing this properties files that means that you know its path :)

Reply:
Same issue that this (https://github.com/telefonicaid/perseo-core/pull/151#discussion_r358873910) , it suggest to extract all URIs to a configuration file.

Reply:
NTC

Reply:
NTC